backend/controllers/project.controller.js

```diff
@@ -67,4 +67,26 @@ ProjectController.destroy = async function (req, res) {
 };
 
 
+ProjectController.getProjectManagers = async function (req, res) {
+  try {
+    const userProjectMap = {};
+    const projects = await Project.find({
+      managedByUsers: { $exists: true, $ne: [] }
+    });
+
+    for (const project of projects) {
+      for (const user of project.managedByUsers) {
+        if (!userProjectMap[user]) {
+          userProjectMap[user] = [];
+        }
+        userProjectMap[user].push(project.name);
+      }
+    }
+    return res.status(200).send(userProjectMap);
```

Comment:
Am I right in understanding that the final out of the array (or object if thats the structure we go with) will only have user IDs? Do we need to make a second db call to populate the user object?